File: 3. CursoPOO/PHP/car.php
<?php
require_once 'account.php'

class Car {
    public $id;
    public $license;
    public $driver;
    public $passegenger;

    public function __construct($license, Account $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar(){
        echo "license: $this->license, 
        conductor: {$this->driver->name},
        document: {$this->driver->document}"
    }
}

File: 3. CursoPOO/Python/car.py
from account import Account

class Car:
    id = int
    license = str
    driver = Account("", "")
    passegenger = int

    def __init__(self, license, driver):
        self.license = license
        self.driver = driver

    def setPassenger(self, passegengerNum):
        if passegengerNum >=4:
            self.__passegenger = int(passegengerNum)
        else:
            print("Necesita asignar 4 pasajeros")

    def getPassenger(self):
        if self.__passegenger != int:
            print(self.__passegenger)

